objects/Wasp/Step_0.gml

```diff
@@ -0,0 +1,83 @@
+/// @description Boss Movement
+// You can write your code in this editor
+
+function wasp_movement(_speed, _bounce_angle) {
```

Comment:
This is redefining this function every frame, safer to define it inside the create event, or better yet in a script to be called on game load 1 time.

Reply:
Changed over to a script.